packages/terra-date-picker/src/react-datepicker/index.jsx

```diff
@@ -801,8 +801,9 @@ class DatePicker extends React.Component {
         isCalendarOpenedViaKeyboard={this.state.isCalendarOpenedViaKeyboard}
         initialTimeZone={this.props.initialTimeZone}
       >
+         <VisuallyHiddenText aria-atomic="true" aria-live="assertive" refCallback={(ref) => { this.visuallyHiddenText = ref; }} className={cx('react-datepicker-visuallyhiddentext')}>
```

Comment:
This change is not valid since it prevents SR from announcing the dates when navigated with arrow keys.

Reply:
Yes Right Sugan Have https://github.com/cerner/terra-framework/pull/2036/commits/50a6cc2ce949626318d6c049d2e645f5bb7f6491 implemented changes.